test/features/step_definitions/settings_steps.rb

```diff
@@ -42,4 +42,10 @@
 
 When(/^I click my avatar$/) do
   find(".settings-link").click
+end
+
+Then(/^I am able to connect to an alternative Trello account$/) do
+  click_link('Connect to a Different Trello Account')
+  step 'I authorize with Trello'
```

Comment:
Just a side note: I'm not a fan of steps calling other steps. Ideally I'd like to pull some of this shared code into some sort of page-object pattern, but was not ready to do this in this PR.